### DISCUSSION TITLE
SainSmart Genmitsu 3018PRO with 5.5W laser

### DISCUSSION THREAD

**Discussion by Geierma** [arkypita/LaserGRBL#866] [Q&A]
Hello, I NEED HELP with using the 5.5W laser bought from SainSmart. The GRBL Laser software is not controlling the laser's on/off function. I have been using the 3018PRO with the 2.5W laser for about a year now with no issues. I attached and plugged in the 5.5W laser and BAM....always on. The 5.5W laser comes with a second controller module that has an on/off switch. I have the stand alone controller also. With the controller connect (not the computer) and the laser module's bottom ON, i can control the laser's ON/OFF with the controller. But once i unplug the controller and switch to the computer the laser is constant ON. How do i get the software to control the laser? M5 command does not work.

**Comment by Geierma** [comment]
I loaded a .nc to the SD card in my controller. Turned the laser module ON. Used the controller to turn the laser off. Then started the .nc file. The laser turned on at the home position, traversed to start point and completed the image, stayed ON while traversing back to home position and remained laser ON.

**Comment by Geierma** [comment]
I plugged in my 2.5W laser and everything worked as designed. Also i used M3 S255 to turn on the laser and M5 to turn it off. Swapping to the 5.5W laser, same as before. Constant on whenever the ON/OFF switch on the external module is on. The software will not control the laser. M5 did not work as it did with the 2.5W laser.

**Comment by Geierma** [comment]
Update #4.........I ran a image that would cause the laser to turn OFF and traverse to a new point and turn ON. Nothing!!! the Laser remains on the entire time. PLEASE HELP!!!!

**Comment by gmmanonymus111** [comment]
Hy, since I couldn't find your actual lasers, I can only guess. Did you buy the correct one? I've seen 3018PRO and 3018PROVer CNC machines and their seperate laser accessories, and they are not compatible with the other machine's accessories (at least that's what the description says). There might be a problem with the TTL frequiency, or somehow the wires are in the wrong order.

> With the controller connect (not the computer) and the laser module's bottom ON, i can control the laser's ON/OFF with the controller. But once i unplug the controller and switch to the computer the laser is constant ON.

I would check the wiring first, as your laser is constantly on.

**Comment by AlanKWatson68** [comment]
I have the same issue with the 5.5W Sainsmart laser did you get a work around?
I've tried a second unit but still doesnt work :(

**Reply by gmmanonymus111** [reply]
Could you send some pictures of your setup/wiring?

**Reply by AlanKWatson68** [reply]
I have tried with the control board powering the laser module directly and with the 12v+ cable removed and with an external PSU.

![20210423_074114](https://user-images.githubusercontent.com/83038882/115830567-63715f00-a408-11eb-92fc-93747c7ecc53.jpg)
Control board connection labels

![20210423_074114](https://user-images.githubusercontent.com/83038882/115830610-73893e80-a408-11eb-85c4-303d342c99a6.jpg)
Cable to board with 12v+ connected
Note PWM wire position

<img width="147" alt="Screenshot 2021-04-23 at 07 46 47" src="https://user-images.githubusercontent.com/83038882/115830660-869c0e80-a408-11eb-9f67-e135fd41814b.png">
Laser board connections

![20210423_074047](https://user-images.githubusercontent.com/83038882/115830766-aa5f5480-a408-11eb-81a8-4b55f755d98e.jpg)
Laser board and cables
Note PWM connector in the middle of the plug

Any help would be great

Cheers
Alan

**Reply by AlanKWatson68** [reply]
Control board connections - correct picture :)
<img width="127" alt="Screenshot 2021-04-23 at 07 46 13" src="https://user-images.githubusercontent.com/83038882/115831001-edb9c300-a408-11eb-9697-350815f3bc79.png">

**Reply by gmmanonymus111** [reply]
I don't think you need another power cable when the GND/PWM/12V cable is plugged in (as this should already provide the necessary electricity). When you tried with different PSU, was it connected to this laser driver(ish) module, or to the control board?
I've found [this doc](https://docs.sainsmart.com/article/wzvu9798tc-genmitsu-3018-pro-5-5-w-laser-module-user-guide) from SainSmart, I don't know if you've already read it, but it could help.

**Comment by Geierma** [comment]
I had to get a new control board. The new laser did not work with the old
board.

**Comment by Geierma** [comment]
Can you send a picture of your control board?

**Comment by AlanKWatson68** [comment]
It’s a:
https://wiki.eleksmaker.com/doku.php?id=eleksmana_se_v3.1

I’ve tried a second board as well….
Only thing I haven’t changed is the cable which I will today

From: Geierma ***@***.***>
Reply to: arkypita/LaserGRBL ***@***.***>
Date: Friday, 23 April 2021 at 08:23
To: arkypita/LaserGRBL ***@***.***>
Cc: AlanKWatson68 ***@***.***>, Comment ***@***.***>
Subject: Re: [arkypita/LaserGRBL] SainSmart Genmitsu 3018PRO with 5.5W laser (#866)

Can you send a picture of your control board?

**Comment by Geierma** [comment]
This is my old board that did *not* work with the 5.5 watt laser.

**Comment by AlanKWatson68** [comment]
So did you change the board?
Or reduce the laser power?
Ive tried it with an external PSU and just use the Mana SE board as the PWM control and still didn’t work ☹

From: Geierma ***@***.***>
Reply to: arkypita/LaserGRBL ***@***.***>
Date: Friday, 23 April 2021 at 08:27
To: arkypita/LaserGRBL ***@***.***>
Cc: AlanKWatson68 ***@***.***>, Comment ***@***.***>
Subject: Re: [arkypita/LaserGRBL] SainSmart Genmitsu 3018PRO with 5.5W laser (#866)

This is my old board that did *not* work with the 5.5 watt laser.

**Comment by AlanKWatson68** [comment]
Ive decided to order a different laser now and try that, thought it should be a simple swap but others have said some lasers just don’t work like that.

From: Alex ***@***.***>
Reply to: arkypita/LaserGRBL ***@***.***>
Date: Friday, 23 April 2021 at 21:28
To: arkypita/LaserGRBL ***@***.***>
Cc: AlanKWatson68 ***@***.***>, Comment ***@***.***>
Subject: Re: [arkypita/LaserGRBL] SainSmart Genmitsu 3018PRO with 5.5W laser (#866)


I don't think you need another power cable when the GND/PWM/12V cable is plugged in (as this should already provide the necessary electricity). When you tried with different PSU, was it connected to this laser driver(ish) module, or to the control board?
I've found this doc<https://docs.sainsmart.com/article/wzvu9798tc-genmitsu-3018-pro-5-5-w-laser-module-user-guide> from SainSmart, I don't know if you've already read it, but it could help.
